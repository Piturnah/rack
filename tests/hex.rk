0xff print
0x42 print
0xffff print