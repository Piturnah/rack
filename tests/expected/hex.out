----STDOUT----
255
66
65535

----STDERR----
